src/index.scss

```diff
@@ -6,18 +6,32 @@
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
-
+*{
+  margin: 0px;
+  padding: 0px;
+  border:none;
+}
+*,
+*::before,
+*::after{
+  box-sizing: border-box;
+}
 
 body {
-  margin: 0;
-  display: flex;
-  place-items: center;
+  
+  // display: flex;
+  // place-items: center;
   min-width: 320px;
   min-height: 100vh;
   background-color: $background-color;
   
 }
-
+.container
+{
```

Comment:
Hmm, interesting, that the bracket is on next line. I wonder if you have code formatter  (like prettier),running on this file, so it is auto formated